src/renderer/components/VideoList.vue

```diff
@@ -6,35 +6,65 @@
      >
        <thumbnail class="thumbnail" :item="item" :mini-player-mode="miniPlayerMode"></thumbnail>
      </div>
+     <div class="pagination-container">
+       <button class="pagination" v-if="prevPageToken" v-on:click="prevPage">prev</button>
```

Comment:
We use [Photon kit](http://photonkit.com/components/) to imitate to maxOS application.
Can you add class attributes `btn btn-default` ?